contrib/osx/keybindings.el

```diff
@@ -30,4 +30,6 @@
                       (interactive)
                       (call-interactively (key-binding "\C-x\C-s"))))
     (global-set-key (kbd "s-Z") 'undo-tree-redo)
-    (global-set-key (kbd "C-s-f") 'spacemacs/toggle-frame-fullscreen)))
+    (global-set-key (kbd "C-s-f") 'spacemacs/toggle-frame-fullscreen)
+    ;; Emacs sometimes registers C-s-f as this weird keycode
+    (global-set-key (kbd "<C-s-268632070>") 'spacemacs/toggle-frame-fullscreen)))
```

Comment:
Beats me, couldn't find much info googling this. But emacs seems to register C-s-f as C-s-268632070 in some cases, for me and others online.